Game/templates/Home/mainpage.html

```diff
@@ -48,8 +48,10 @@
     <div class="container">
       <div class="row">
         <div class="col-lg-12 text-center">
-          <h2 class="section-heading text-uppercase">How to play <img src="{% static 'Base/img/logo.png'%}" width="28" height="28"> </h2>
-          <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
```

Comment:
It's ok if you want to change it to the scoreboard.

Reply:
Yes, because actually our game is kinda easy to play.